src/order-history/OrderHistoryPage.jsx

```diff
@@ -84,7 +84,7 @@ class OrderHistoryPage extends React.Component {
   renderOrdersTable() {
     return (
       <Table
-        className="order-history"
+        className="order-history table-bordered"
```

Comment:
Will this change something, or is it to keep it the same?

Reply:
Keep it the same – we updated the default style of tables to not be bordered to provide more flexibility.